File: cinema_upload/src/main/webapp/js/service/typeService.js
//服务层
app.service('typeService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){

		return $http.get('./type/findAll.do');
	}
	//分页 
	this.findPage=function(page,rows){
		return $http.get('../type/findPage.do?pageNum='+page+'&pageSize='+rows);
	}
	//查询实体
	this.findOne=function(id){
		return $http.get('../type/findOne.do?id='+id);
	}
	//增加 
	this.add=function(entity){
		return  $http.post('../type/add.do',entity );
	}
	//修改 
	this.update=function(entity){
		return  $http.post('../type/update.do',entity );
	}
	//删除
	this.dele=function(ids){
		return $http.get('../type/delete.do?ids='+ids);
	}
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('../type/search.do?pageNum='+page+"&pageSize="+rows, searchEntity);
	}    	
});

File: client_manage_interface/src/main/java/cn/edu/aynu/rjxy/management/service/TypeService.java
package cn.edu.aynu.rjxy.management.service;


import cn.edu.aynu.rjxy.pojo.TbType;
import com.github.pagehelper.PageInfo;

import java.util.List;

/**
 * 服务层接口
 * @author Administrator
 *
 */
public interface TypeService {

	/**
	 * 返回全部列表
	 * @return
	 */
	public List<TbType> findAll();
	
	
	/**
	 * 返回分页列表
	 * @return
	 */
	public PageInfo findPage(int pageNum, int pageSize);


	/**
	 * 增加
	*/
	public void add(TbType type);


	/**
	 * 修改
	 */
	public void update(TbType type);


	/**
	 * 根据ID获取实体
	 * @param id
	 * @return
	 */
	public TbType findOne(Long typeId);


	/**
	 * 批量删除
	 * @param ids
	 */
	public void delete(Long[] typeIds);

	/**
	 * 分页
	 * @param pageNum 当前页 码
	 * @param pageSize 每页记录数
	 * @return
	 */
	public PageInfo findPage(TbType type, int pageNum, int pageSize);
	
}
